hazelcast/src/main/java/com/hazelcast/dataconnection/databasediscovery/impl/MySQLDatabaseDiscovery.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.dataconnection.databasediscovery.impl;
+
+import com.hazelcast.dataconnection.DataConnectionResource;
+import com.hazelcast.dataconnection.impl.JdbcDataConnection;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import static com.hazelcast.dataconnection.impl.JdbcDataConnection.OBJECT_TYPE_TABLE;
+
+public class MySQLDatabaseDiscovery {
+
+    List<String> systemSchemaList = List.of("sys");
```

Comment:
This is not used.

Reply:
rev : ccb44d785f9dae01e17a33838395b82cc2260434